src/deluge/dsp/filter/filter.h

```diff
@@ -81,7 +81,7 @@ class Filter {
 
 		//this is 1q31*1q16/(1q16+tan(f)/2)
 		//tan(f) is q17
-		divideBy1PlusTannedFrequency = (int64_t)ONE_Q31U * ONE_Q16 / (ONE_Q16 + (tannedFrequency >> 1));
+		divideBy1PlusTannedFrequency = (q31_t)(288230376151711744.0f / (float)(ONE_Q16 + (tannedFrequency >> 1)));
```

Comment:
I'm going to look at this all holistically, I suspect the whole setup routine can be simplified to use float math to calculate the coefficients